### PR TITLE
make it works on RN 0.29

### PR DESCRIPTION
**Pull request by arthurlee**
Fix some minus errors.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,4 +6,6 @@ import ActionSheetAndroid from './ActionSheetAndroid';
 
 var ActionSheet = Platform.OS === 'ios' ? ActionSheetIOS : ActionSheetAndroid;
 
-export default ActionSheet;
+//export default ActionSheet;
+
+module.exports = ActionSheet;
```
Comment (martincik) [inline index.js]: This is ES5 equivalent of what I've got in the past. Can you revert this line back to ES6 style? I'll then merge this PR. Thank you.

Comment (arthurlee) [inline index.js]: I do that because it does not work for me. That's strange. So I change to ES5 syntax. I'm using react-native 0.29.2.

Comment (arthurlee) [inline index.js]: Any suggestion? Thanks.

**Comment by martincik** [comment]
👍 💯  Thank you.

**Pull request merged**